app/views/layouts/_header.html.erb

```diff
@@ -120,7 +122,7 @@
                 <li><a href="/logoutRemotely">Logout from all devices</a></li>
               </ul>
             <% else %> <!-- Login button -->
-              <a data-toggle="modal" data-target="#loginModal">
+              <a data-toggle="modal" data-target="#userModal" data-mode="login">
```

Comment:
let's name it login modal because many prs are linked with login modal. And we always want the login modal to pop up. User modal seems to look ambiguous to me

Reply:
Actually we need the loginmodal to be displayed always. Only in 1% situation we will be using the signup modal. That is when the user first signs up at Public Lab. Then the users will never use it again.
They will be using the login modal. Does this makes sense?

Reply:
Ok. I've pushed changes that renamed the modals.

Reply:
Please change this back to login modal.

Reply:
Ok great

Reply:
This is how I've managed to get it working on my new version:
data-toggle="modal" data-target="#loginModal" data-mode="signup"

In the data-mode section field, you specify either login or signup

Reply:
Looks cool